tensorflow_serving/batching/batching_util.h

```diff
@@ -0,0 +1,206 @@
+/* Copyright 2017 Google Inc. All Rights Reserved.
```

Comment:
The only function that needs to be exposed in the API here is AddPadding() right? Can you move the others to the .cc file and out of the public interface?

As part of this, the unit test would ideally only test AddPadding() (and cover the others indirectly).